test/CMakeLists.txt

```diff
@@ -74,13 +74,13 @@ endif()
 
 
 
-find_package(HepMC)
+find_package(HepMC3)
 find_package(HepPDT)
 
-if(HepMC_FOUND AND HepPDT_FOUND )
+if(HepMC3_FOUND AND HepPDT_FOUND )
   add_executable(edm4hep_testhepmc hepmc/edm4hep_testhepmc.cc)
-  target_include_directories(edm4hep_testhepmc PUBLIC ${HEPMC_INCLUDE_DIR} ${HEPPDT_INCLUDE_DIR} )
-  target_link_libraries(edm4hep_testhepmc edm4hep podio::podioRootIO ${HEPPDT_LIBRARIES} ${HEPMC_LIBRARIES})
+  target_include_directories(edm4hep_testhepmc PUBLIC ${HEPMC3_INCLUDE_DIR} ${HEPPDT_INCLUDE_DIR} )
+  target_link_libraries(edm4hep_testhepmc edm4hep podio::podioRootIO ${HEPPDT_LIBRARIES} ${HEPMC3_LIBRARIES})
```

Comment:
Minute detail here: Does HepMC3 export targets that could be used here?

Reply:
Sadly no, see the upstream example: https://gitlab.cern.ch/hepmc/HepMC3/-/blob/master/examples/BasicExamples/CMakeLists.txt

Reply:
Ah, pity.